nebula/ui/components/MZCheckBoxRow.qml

```diff
@@ -64,7 +64,7 @@ RowLayout {
             Layout.fillWidth: true
             text: subLabelText
             visible: !!subLabelText.length
-            wrapMode: Text.WordWrap
+            wrapMode: Text.Wrap
```

Comment:
Why not `WrapAtWordBoundaryOrAnywhere`?

Reply:
They appear to be the same thing, though `WrapAtWordBoundaryOrAnywhere` is [not documented](https://doc.qt.io/qt-6.2/qml-qtquick-text.html#wrapMode-prop) for use in QML

Reply:
Thanks, sorry for missing that documentation.